nvm.sh

```diff
@@ -138,6 +138,36 @@ nvm_download() {
   fi
 }
 
+#################################################################################
+#  The software package download function has the same function as nvm_download.
+#  this function will return the HTTP status after download file.If the download is successful,
+#  return 200. Otherwise, return different status codes based on the results
+#  return: http code
+#################################################################################
+nvm_download_v2() {
```

Comment:
is there a reason to keep the original nvm_download? iow, why not change the existing implementation?

Reply:
thanks for your review, i already fix that.